cli/Sources/TuistServer/Client/ServerAuthenticationController.swift

```diff
@@ -74,92 +88,175 @@ public struct ServerAuthenticationController: ServerAuthenticationControlling {
         #endif
     }
 
+    public func refreshToken(serverURL: URL, inSubprocess: Bool) async throws {
+        try await authenticationTokenRefreshingIfNeeded(
+            serverURL: serverURL,
+            forceRefresh: true,
+            inSubprocess: inSubprocess
+        )
+    }
+
     public func refreshToken(serverURL: URL) async throws {
         try await authenticationTokenRefreshingIfNeeded(
-            serverURL: serverURL, forceRefresh: true
+            serverURL: serverURL,
+            forceRefresh: true,
+            inSubprocess: true
         )
     }
```

Comment:
I'd change the default to be `false` given we use this component for iOS and macOS app, too.